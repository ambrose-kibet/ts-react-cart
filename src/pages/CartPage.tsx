import CartContainer from '../components/CartContainer';

const CartPage = () => {
  return <CartContainer />;
};
export default CartPage;
