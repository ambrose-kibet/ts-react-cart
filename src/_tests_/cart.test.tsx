import { render } from '@testing-library/react';
import { it, describe, expect } from 'vitest';
import CartProvider from '../context/CartContext';
import CartPage from '../pages/CartPage';

describe('cart page test', () => {
  it('should render correctly', () => {
    const { container } = render(
      <CartProvider>
        <CartPage />
      </CartProvider>
    );
    expect(container).toMatchSnapshot();
  });
});
