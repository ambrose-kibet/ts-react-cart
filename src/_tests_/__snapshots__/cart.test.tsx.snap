// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`cart page test > should render correctly 1`] = `
<div>
  <h2
    style="text-align: center;"
  >
    Your Cart is Empty
  </h2>
</div>
`;
